tests/unit/executor/test_step_handler.py

```diff
@@ -325,7 +326,7 @@ def setup(self, new_dir, partitions):
 
     def test_run_scriptlet(self, new_dir, capfd):
         sh = _step_handler_for_step(
-            Step.PULL,
+            Step.BUILD,
```

Comment:
how come?

Reply:
This test was running a build step scriptlet in the pull step. Not a problem for what was being tested, but it was a non-realistic, inconsistent scenario.